src/pages/Troubleshooting.vue

```diff
@@ -94,6 +107,15 @@ export default {
     showLogs() {
       ipcRenderer.send('troubleshooting/show-logs');
     },
+    restart() {
+      this.state = K8s.State.STOPPING;
+      ipcRenderer.send('k8s-restart');
+    },
+    updateDebug(value) {
+      console.log(`setting debug mode to ${ value }`);
+      ipcRenderer.invoke('settings-write', { debug: value })
+        .then(() => this.restart());
```

Comment:
I can't decide if restarting here is a good thing — if the cluster is working fine, do we need to bring it down? But I guess if the user is flipping switches it's _probably_ not working fine?